.env.example

```
@@ -6,7 +6,20 @@ DOCKER_DATABASE_NAME="startui"
 DOCKER_DATABASE_USERNAME="startui"
 DOCKER_DATABASE_PASSWORD="startui"
 
+DOCKER_MINIO_API_PORT="9000"
+DOCKER_MINIO_UI_PORT="9001"
+DOCKER_MINIO_USERNAME="minioadmin"
+DOCKER_MINIO_PASSWORD="minioadmin"
+
+# S3
+S3_ENDPOINT="http://127.0.0.1:${DOCKER_MINIO_API_PORT}"
+S3_BUCKET_NAME="default"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)